notebooks/en/enterprise_cookbook_argilla.ipynb

```diff
@@ -0,0 +1,1404 @@
+{
```

Comment:
<p>Also for more clarity when discussing "Inference API".</p><p> </p><p>"Now we can make a standard API request to the Serverless Inference API (docs). Note that the Serverless Inference API is mostly for testing and is rate limited. For testing without rate limits, you can create your own API via the HF Dedicated Inference Endpoints."</p><p> </p><p>A tip callout would be easier to see I think</p><p> </p><p>&gt; [!TIP]</p><p>&gt; The code below will be updated once the Inference API recipe is finished.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>updated accordingly. the [!TIP] does not display properly in my local vs code markdown, but hopefully is displays correctly on the website</p>